static/js/popovers.js

```diff
@@ -508,34 +398,16 @@ export function render_actions_remind_popover(element, id) {
             trigger: "manual",
         });
         elt.popover("show");
-        current_flatpickr_instance = $(
-            `.remind.custom[data-message-id="${CSS.escape(message.id)}"]`,
-        ).flatpickr({
+        $(`.remind.custom[data-message-id="${CSS.escape(message.id)}"]`).flatpickr({
```

Comment:
We're apparently removing the tracking of `flatpickr` instances; can that lead to them being leaked / not destroyed?

Reply:
`current_flatpickr_instance` is not being used anywhere. This change actually saves memory, I suppose.

Reply:
OK